experiments/scripts/subj-scripts/crystal.sh

```diff
@@ -0,0 +1,19 @@
+# The root directory for the old subject
+export DT_SUBJ_ROOT=/home/user/dependent-tests-impact/experiments/crystalvc
+# Directory for where all of the old subject's information is stored.
+export DT_SUBJ=$DT_SUBJ_ROOT
+# Directory containing the old subject's src directory.
+export DT_SUBJ_SRC=$DT_SUBJ_ROOT
+
+# The root directory for the new subject
+export NEW_DT_SUBJ_ROOT=/home/user/dependent-tests-impact/experiments/crystal
+# Directory for where all of the new subject's information is stored.
+export NEW_DT_SUBJ=$NEW_DT_SUBJ_ROOT
```

Comment:
$DT_SUBJ, $DT_SUBJ_SRC, $NEW_DT_SUBJ and $NEW_DT_SUBJ_SRC all seem to be set to the same variables and therefore should be moved to the setup-vars-ant.sh